include/ibamr/IBFESurfaceMethod.h

```diff
@@ -521,6 +568,16 @@ class IBFESurfaceMethod : public IBStrategy
 
     /*
      * FE data associated with this object.
+     * d_X_systems: IB coordinates system
```

Comment:
Probably can drop `IB` here:

```suggestion
     * d_X_systems: coordinates system
```

etc.